Neos.Neos/Classes/Command/UserCommandController.php

```diff
@@ -317,7 +370,7 @@ public function removeRoleCommand($username, $role, $authenticationProvider = nu
      * @return User The user
      * @throws Exception
      */
-    protected function getUserOrFail($username, $authenticationProviderName)
+    protected function getUserOrFail($username, $authenticationProviderName = null)
```

Comment:
This seems unused now?

Reply:
It was still used in setPassword. But you are right it makes no sense to keep it as a seperate method. I refactored that.